app/routes/routes.go

```diff
@@ -1,20 +1,17 @@
 package routes
 
 import (
+	"star-pos/app/middlewares"
 	userHandler "star-pos/features/user/handler"
-	userData "star-pos/features/user/repository"
-	userService "star-pos/features/user/service"
-	encrypts "star-pos/utils"
 
 	"github.com/labstack/echo/v4"
-	"gorm.io/gorm"
 )
 
-func InitRouter(e *echo.Echo, db *gorm.DB) {
-	userData := userData.New(db)
-	hashData := encrypts.NewHashService()
-	userService := userService.New(userData, hashData)
-	userHandler := userHandler.New(userService)
+func InitRouter(e *echo.Echo) {
 	e.GET("/hello", userHandler.Hello)
 	e.POST("/users", userHandler.CreateAccount)
+	e.POST("/login", userHandler.Login)
+	e.GET("/users", userHandler.GetProfile, middlewares.JWTMiddleware())
+	e.PATCH("/users", userHandler.UpdateProfile, middlewares.JWTMiddleware())
```

Comment:
please read the task requirement in the task description carefully. this should be PATCH /users/{id}. which we update the user based on the user ID, not JWT